packages/app-account/src/components/ProfileForm.js

```diff
@@ -133,15 +133,15 @@ class ProfileForm extends Component {
 
     if (isEditable) {
       return (
-        <div>
+        <div id="ib-profile">
```

Comment:
Can we make these ids specific to Selenium testing, e.g. ib-e2e-profile, so there's no confusion about what these ids are being used for?

Reply:
I'd recommend using a term like 'test' instead in this case.